docs/content/en/main/concepts/authorization.md

```diff
@@ -43,26 +43,24 @@ They are related in the following way:
 4. one of the local authorizer or bootstrap policy authorizer must allow.
 
 ```
-                                                                                          ┌──────────────┐
-                                                                                          │              │
-           ┌─────────────────┐     ┌───────────────┐        ┌───────────────────┐   ┌────►│ Local Policy ├──┐
-           │ Top-level       │     │               │        │                   │   │     │ authorizer   │  │
- request   │ Organization    │     │ Workspace ┌───┴───┐    │ Max. Permission   │   │     │              │  │
-──────────►│ authorizer      ├────►│ Content   │+groups├───►│ Policy authorizer ├───┤     └──────────────┘  │
-           │                 │     │ authorizer└───┬───┘    │                   │   │                       ▼
-           │                 │     │               │        └───────────────────┘   │                       OR───►
-           └─────────────────┘     └───────────────┘                                │     ┌──────────────┐  ▲
-                                                                                    │     │  Bootstrap   │  │
-                                                                                    └────►│  Policy      ├──┘
-                                                                                          │  authorizer  │
-                                                                                          │              │
-                                                                                          └──────────────┘
-
+                                                                                                            ┌──────────────┐
+                                                                                                            │              │
+           ┌─────────────────┐     ┌───────────────┐        ┌─────────────┐   ┌───────────────────┐   ┌────►│ Local Policy ├──┐
+           │ Top-level       │     │               │        │ Reverse     │   │                   │   │     │ authorizer   │  │
+ request   │ Organization    │     │ Workspace ┌───┴───┐    │ Permission  │   │ Max.BPermission   │   │     │              │  │
```

Comment:
```suggestion
 request   │ Organization    │     │ Workspace ┌───┴───┐    │ Permission  │   │ Max. Permission   │   │     │              │  │
```

Reply:
oh dear this online tool is buggy :-/ fixed